Source/WebKit/UIProcess/TextAnimationType.h

```diff
@@ -37,7 +37,8 @@ enum class TextAnimationType : uint8_t {
 
 struct TextAnimationData {
     TextAnimationType style;
-    WTF::UUID remainingRangeUUID { WTF::UUID::emptyValue };
+    WTF::UUID unanimatedRangeUUID { WTF::UUID::emptyValue };
```

Comment:
Is `{ WTF::UUID::emptyValue }` really necessary?

Reply:
TextAnimationType.h:40:35: error: field of type 'WTF::UUID' has private default constructor
   40 |     WTF::UUID unanimatedRangeUUID { };

So, yes.